seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-webhook/src/main/scala/org/apache/seatunnel/spark/webhook/source/Webhook.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.spark.webhook.source
+
+import org.apache.seatunnel.spark.SparkEnvironment
+import org.apache.seatunnel.spark.stream.SparkStreamingSource
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.{Dataset, Row, SparkSession, SQLContext}
+import org.apache.spark.streaming.dstream.DStream
+
+class Webhook extends SparkStreamingSource[String] {
+
+  override def start(env: SparkEnvironment, handler: Dataset[Row] => Unit): Unit = {
+    var spark = env.getSparkSession
+    // Create HTTP Server and start streaming
+    implicit val sqlContext: SQLContext = spark.sqlContext
+
+    var port = if (config.hasPath("port")) config.getInt("port") else 9999
+    var baseUrl = if (config.hasPath("path")) config.getString("path") else "/"
+
+    val query = new JettyServerStream(port, baseUrl)
+      .toDF
+      .writeStream
+      .foreachBatch((batch, batchId) => {
+        handler(batch)
+      })
+      .start()
+
+    query.awaitTermination()
+  }
+
+  override def rdd2dataset(sparkSession: SparkSession, rdd: RDD[String]): Dataset[Row] = { null }
+
+  override def getData(env: SparkEnvironment): DStream[String] = { null }
```

Comment:
You need to implement this method, otherwise the data received by http datasource will not be processed by downstream plugin

Reply:
I rewrote the start method, and getData and rdd2dataset are used in the start method, so there is no need to implement these two methods here.

Reply:
Ok, I understand. It's OK to rewrote, but it might be better only implement the `getData` and `rdd2dataset`, you don't need to break this design.

Reply:
I am not very familiar with spark and spark stream, and the current implementation is referring to other demos. For what you said, split into the getData and rdd2dataset, I didn't find a usable implementation to do it. Can you provide reference code for implementation.

Reply:
You can find a demo in `KafkaStream`, but this may not be a key point need to change in this PR. It's OK for me to rewrite the `start` method. I will close this conversation.